### PR TITLE
Jh 100 team

### PR DESCRIPTION
**Pull request by kisto**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (Sebosek) [inline package.json:1]: Should be part of web-jehlomat, I guess.

File: web-jehlomat/src/config/user.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import jwt_decode from "jwt-decode";
+import { AxiosResponse } from 'axios';
+import {API} from './baseURL';
+
+interface IToken {
```
Comment (Sebosek) [inline web-jehlomat/src/config/user.ts:5]: Already defined in `types.ts`

File: web-jehlomat/src/config/user.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    "aud": string;
+    "user-id": string;
+    "iss": string;
+    "exp": string;
+}
+
+export interface IResponse {
```
Comment (Sebosek) [inline web-jehlomat/src/config/user.ts:12]: Same here.

File: web-jehlomat/src/screens/Team/components/ConfirmModal.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    open: boolean;
+    close: any;
+    children?: React.ReactNode;
+    isEdit: boolean;
+}
+
+export const ConfirmModal: React.FC<any> = (props: React.PropsWithChildren<ModalProps>) => {
+
+    return (
+        <>
```
Comment (Sebosek) [inline web-jehlomat/src/screens/Team/components/ConfirmModal.tsx:14]: Fragment is not needed here.

File: web-jehlomat/src/screens/Team/components/MapModal.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    open: boolean;
+    close: any;
+    children?: React.ReactNode;
+}
+
+/* Go to page on mobile, show modal on desktop */
+export const MapModal: React.FC<any> = (props: React.PropsWithChildren<ModalProps>) => {
+
+    return (
+        <>
```
Comment (Sebosek) [inline web-jehlomat/src/screens/Team/components/MapModal.tsx:14]: Fragment is not needed here.

File: web-jehlomat/src/types.ts [hunk 6]
```diff
@@ -19,3 +19,13 @@ export interface IUser {
     isAdmin: boolean;
 }
 
+export interface IUserEdited {
```
Comment (Sebosek) [inline web-jehlomat/src/types.ts:22]: Is it really necessary to have special interface for user update? Can't we merge it with already existing User type? I mean, if I set all the non-required props to undefined I should get undefined values also for read operation.

**Review by Sebosek — COMMENTED** [review]
Please lint the code.

File: web-jehlomat/src/utils/links.ts [hunk 7]
```diff
@@ -0,0 +1,23 @@
+export const LINKS = {
```
[the remark this answers is not in the document]
Comment (Daidalos117) [reply web-jehlomat/src/utils/links.ts:1]: seems like old LINKS object, should be imported from routes.ts quite a while now..

File: web-jehlomat/src/config/user.ts [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    teamId: number;
+    isAdmin?: boolean;
+    verified?: boolean;
+}
+
+export const getMe = (token: string) => {
+    const decoded: IToken = jwt_decode(token);
+    return decoded["user-id"];
+}
+export const getUser = async (token: string) => {
```
Comment (Daidalos117) [inline web-jehlomat/src/config/user.ts:27]: this whole file shouldn't be needed at the moment, you can use user that is already fetched from state like this `const user = useRecoilValue(userState);`

File: web-jehlomat/src/utils/links.ts (hunk 7 above)
[the remark this answers is not in the document]
Comment (kisto) [reply web-jehlomat/src/utils/links.ts:1]: not sure what is the problem here...

File: package.json (hunk 1 above)
Comment (kisto) [reply package.json:1]: ?

File: web-jehlomat/src/config/user.ts (hunk 2 above)
Comment (kisto) [reply web-jehlomat/src/config/user.ts:5]: deleted

File: web-jehlomat/src/types.ts (hunk 6 above)
Comment (kisto) [reply web-jehlomat/src/types.ts:22]: API accept different payload. So I prefer to keep it separated

File: package.json (hunk 1 above)
Comment (Sebosek) [reply package.json:1]: Just remove the package.json file from the root of the project.

File: web-jehlomat/src/utils/links.ts (hunk 7 above)
[the remark this answers is not in the document]
Comment (Sebosek) [reply web-jehlomat/src/utils/links.ts:1]: Dead code elimination

File: web-jehlomat/src/types.ts (hunk 6 above)
Comment (Sebosek) [reply web-jehlomat/src/types.ts:22]: @fechtner please have a look, probably Swagger is wrongly cpecified.

File: web-jehlomat/src/utils/links.ts (hunk 7 above)
[the remark this answers is not in the document]
Comment (Daidalos117) [reply web-jehlomat/src/utils/links.ts:1]: it's just deprecated.... if you need to this object, import it from routes.ts. Nevertheless this file should be deleted.

[the remark this answers is not in the document]
Comment (kisto) [reply web-jehlomat/src/utils/links.ts]: thanks!

**Pull request merged**